pkg/yang/entry.go

```diff
@@ -943,6 +947,30 @@ func ToEntry(n Node) (e *Entry) {
 	return e
 }
 
+// addExtrasToLeaf stores the values for unimplemented keywords in leaf entries.
+func addExtrasToLeaf(n Node, e *Entry) {
```

Comment:
Slightly pedantically, perhaps `addExtraKeywordsToLeafEntry`?  

Mostly to avoid someone coming along later and saying "oh, I can add more extra stuff here", when this mightn't be the right place for another use case.

Reply:
Good call. Done.